foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePythonPropertiesFileMigration.java

```diff
@@ -0,0 +1,119 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.replaceLiteralInFile;
+
+public class AiopsReferencePythonPropertiesFileMigration extends AbstractAissembleMigration {
+
+    public static final Logger logger = LoggerFactory.getLogger(AiopsReferencePythonPropertiesFileMigration.class);
+    public static final Map<String, String> AIOPS_REFERENCE_PYTHON_PROPERTIES_FILE_PACKAGE_MAP = Map.of(
+            "policy-decision-point.aiops-security.properties.vm", "policy-decision-point.aissemble-security.properties.vm"
+            );
+
+    /**
+     * Function to check whether the migration is necessary.
+     * @param file file to check
+     * @return shouldExecute - whether the migration is necessary.
+     */
+    @Override
+    protected boolean shouldExecuteOnFile(File file) {
+        boolean shouldExecute = false;
+
+//        getMavenProject().getPlugin("");
+
+        try {
+            if(file.getName().equals("aiops-security.properties")) {
+                BufferedReader aiopsReferencePythonPropertiesFileConfig = new BufferedReader((new FileReader(file)));
+
+                String lineOne;
+                while((lineOne = aiopsReferencePythonPropertiesFileConfig.readLine()) !=null && !shouldExecute) {
+                    for (String key : AIOPS_REFERENCE_PYTHON_PROPERTIES_FILE_PACKAGE_MAP.keySet()) {
+                        if (lineOne.contains(key)) {
+                            return true;
```

Comment:
Would be easier to read if instead of returning, we set the shouldExecute to true and have just one return statement